modules/nw-networkoperator-crd.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-network-operator.adoc
+
+[id="network-networkoperator-default-crd-{context}"]
+= Network Operator custom resource
+
+The `NetworkConfig` configuration in a `networkoperator.openshift.io` Custom
+Resource (CR) object stores the advanced settings for the `Network Operator`.
+
+The following parameters are set in the `config.openshift.io` CR and the
+installation program copies them to the `networkoperator.openshift.io` CR object
+before the cluster is created.
+
+* `cidr`
+* `hostPrefix`
+* `networkType`
+* `serviceNetwork`
+
+The following CR displays the default configuration for the Network Operator and
+explains both the parameters you can configure and valid parameter values:
+
+.Default NetworkConfig CR
+[source,yaml]
+----
+apiVersion: networkoperator.openshift.io/v1
+kind: NetworkConfig
+spec:
+  clusterNetworks:
+  - cidr: 10.128.0.0/14 <1>
+    hostSubnetLength: 9 <2>
+  serviceNetwork: 172.30.0.0/16 <3>
+  defaultNetwork: <4>
+    type: OpenShiftSDN <5>
+    openshiftSDNConfig:
+      mode: NetworkPolicy <6>
+      mtu: 1450 <7>
+      vxlanPort: 4789 <8>
+  deployKubeProxy: false <9>
+  kubeProxyConfig: <10>
+    bindAddress: 0.0.0.0 <11>
+    iptablesSyncPeriod: 30s <12>
+    proxyArguments: <13>
+      iptables-min-sync-period:
+      - 30s
+----
+
+<1> `cidr`: A block of IP addresses from which pod IPs are allocated.
+{product-title} uses the Classless Inter-Domain Routing (CIDR) standard to
+allocate IP addresses. The 'OpenShiftSDN' network plugin supports multiple
+ClusterNetworks. The address blocks for multiple ClusterNetworks must not
+overlap. This parameter is set in the `config.openshift.io` object.
+<2> `hostPrefix`: The address size to assign to each individual node. When a new
+node is added to the cluster, 2^(32-`hostPrefix`) addresses are allocated: when
+`hostPrefix` is set to /23, each node has 512 pod IP addresses. This parameter
+is set in the `config.openshift.io` object.
+<3> `serviceNetwork`: A block of IP addresses for services. `OpenShiftSDN`
+allows only one serviceNetwork block. This parameter is set in the
+`config.openshift.io` object.
+<4> `defaultnetwork`: The header for the parameters that specify the
+default pod network.
+<5> `NetworkType`: The Network plugin that is to be deployed. `OpenShiftSDN` is
+the only plugin supported in {product-name} {product-version}. This parameter is
+set in the `config.openshift.io` object.
+<6> `mode`: Configures the isolation mode for `OpenShiftSDN`. The allowed values
+are `Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is
+`NetworkPolicy`.
+<7> `mtu`: The Maximum Transmission Unit (MTU) for the VXLAN overlay network.
+The default value is the MTU of the node that the cluster-network-operator is first
+run on, minus 50 bytes for overhead. If the nodes in your cluster do not have
+the same MTU, then you must set this explicitly.
```

Comment:
Talking about "the node that the CNO is first run on" may be too much info. "This is normally configured automatically, but if the nodes in your cluster do not all have the same MTU, then you must set this explicitly, to 50 less than the smallest node MTU value."